extensions/smallrye-reactive-messaging-amqp/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/amqp/deployment/AmqpDevServicesBuildTimeConfig.java

```diff
@@ -32,13 +32,13 @@ public class AmqpDevServicesBuildTimeConfig {
      *
      * Check https://quay.io/repository/artemiscloud/activemq-artemis-broker to find the available versions.
      */
-    @ConfigItem(defaultValue = "quay.io/artemiscloud/activemq-artemis-broker:0.1.2")
+    @ConfigItem(defaultValue = "quay.io/artemiscloud/activemq-artemis-broker:1.0.11")
     public String imageName;
 
     /**
      * The value of the {@code AMQ_EXTRA_ARGS} environment variable to pass to the container.
      */
-    @ConfigItem(defaultValue = "--no-autotune --mapped --no-fsync")
+    @ConfigItem(defaultValue = "--no-autotune --mapped --no-fsync --relax-jolokia --http-host 0.0.0.0")
```

Comment:
For my own curiosity: what does relax-jolokia do?

Reply:
By default Artemis restricts the list or allowed origins to access the jolokia console

```xml
<allow-origin>*://0.0.0.0*</allow-origin>
```

This flag removes the restriction to allow external connections to the container.